packages/page-claims/src/index.tsx

```diff
@@ -164,19 +160,15 @@ function ClaimsApp (): React.ReactElement {
 
   // If it's 1/ not preclaimed and 2/ not the old claiming process, fetch the
   // statement kind to sign.
-  const statementKind = useCall<string>(!isPreclaimed && !isOldClaimProcess && ethereumAddress && api.query.claims.signing, [ethereumAddress], {
+  const _statementKind = useCall<string>(!isPreclaimed && !isOldClaimProcess && ethereumAddress && api.query.claims.signing, [ethereumAddress], {
     transform: (option: Option<StatementKind>) => option.unwrapOr('').toString()
   });
-
-  const statement = statementKind
-    ? statementKind === ALTERNATIVE_STATEMENT
-      ? statementAlternative
-      : statementRegular
-    : null;
+  const statementKind = api.createType('StatementKind', _statementKind);
```

Comment:
Would recommend, in the transform, to do `option.unwrapOr(null)`. Then you won't need this last line, and `statementKind` will have type `StatementKind | null`

Reply:
Thanks, that works. Because of the `useCall` it's `StatementKind | undefined | null` though.